build.gradle

```diff
@@ -614,7 +614,8 @@ artifactoryPublish {
 def dockerVariants = [
   "openjdk-11",
   "graalvm",
-  "openjdk-latest"
+  "openjdk-latest",
+  "openjdk-11-debug"
```

Comment:
just an esthetic suggestion, what about moving it just below "openjdk-11" to have a nicer grouping?